nodebuilder/header/service.go

```diff
@@ -13,6 +13,9 @@ import (
 	modfraud "github.com/celestiaorg/celestia-node/nodebuilder/fraud"
 )
 
+// ErrHeightNegative returned when the provided block height is <= 0.
+var ErrHeightNegative = errors.New("height is negative")
```

Comment:
Let's call it ErrInvalidHeight and say height is invalid when <= 0

Reply:
changed heights to uint64 to be consistent with your repo and switched to ErrHeightZero: [8bc6a66](https://github.com/celestiaorg/celestia-node/pull/3470/commits/8bc6a664b4c2c9ad1220d0b42abd9c71d6f82822)